Student/Student/Routes.swift

```diff
@@ -95,7 +95,14 @@ let routeMap: KeyValuePairs<String, RouteHandler.ViewFactory?> = [
         if assignmentID == "syllabus" {
             return StudentSyllabusViewController.create(courseID: ID.expandTildeID(courseID))
         }
-        if let controller = moduleItemController(for: url, courseID: courseID) { return controller }
```

Comment:
don't we still need this if the flag is disabled?

Reply:
Good catch